File: 445-Assign Cookies-分发饼干/445-Assign Cookies-分发饼干.java
class Solution {
    public int findContentChildren(int[] g, int[] s) {
        Arrays.sort(g);
        Arrays.sort(s);

        int gPoint = 0;
        int sPoint = 0;
        int count = 0;

        while(gPoint < g.length && sPoint < s.length){
            if(g[gPoint] <= s[sPoint]){
                gPoint++;
                count++;
            }
            sPoint++;
        }
        return count;
    }
}

File: 445-Assign Cookies-分发饼干/445-Assign Cookies-分发饼干.js
/**
 * @param {number[]} g
 * @param {number[]} s
 * @return {number}
 */
 var findContentChildren = function(g, s) {
    g.sort((a, b) => a-b)
    s.sort((a, b) => a-b)
    let gPint = 0
    let sPoint = 0
    let count = 0
    while(gPint < g.length && sPoint < s.length){
        if(g[gPint] <= s[sPoint]){
            gPint++
            count++
        }
        sPoint++
    }
    return count
};